tests/unit/client/feedback/schemas/test_questions.py

```diff
@@ -108,6 +133,28 @@ def test_label_question_errors(
                 "visible_options": None,
             },
         ),
+        (
+            {"name": "a", "description": "a", "required": True, "labels": list(string.ascii_lowercase)},
+            {
+                "type": "label_selection",
+                "options": [{"value": a, "text": a} for a in string.ascii_lowercase],
+                "visible_options": None,
```

Comment:
In terms of readability, I think it would be best to use a numeric range as sample data:
```python
..."labels": [str(i) for i in range(0, 21)],
{..., "options" : [{"value": str(i), "text": str(i)} for i range(0,21)],
```

Reply:
Sure, I'll change that!